src/current/cockroachcloud/export-logs.md

```diff
@@ -377,6 +379,117 @@ Perform the following steps to enable log export from your CockroachDB {{ site.d
 Exporting Logs to Azure Monitor from a CockroachDB {{ site.data.products.dedicated }} cluster is in **[limited access](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/cockroachdb-feature-availability)** and is only available to enrolled organizations. To enroll your organization, contact your Cockroach Labs account team. This feature is subject to change.
 {{site.data.alerts.end}}
 
+Perform the following steps to enable log export from your CockroachDB {{ site.data.products.dedicated }} cluster to Azure Monitor.
+
+1. Find your CockroachDB {{ site.data.products.dedicated }} cluster ID:
+
+	1. Visit the CockroachDB {{ site.data.products.cloud }} console [cluster page](https://cockroachlabs.cloud/clusters).
+	1. Click on the name of your cluster.
+	1. Find your cluster ID in the URL of the single cluster overview page: `https://cockroachlabs.cloud/cluster/{your_cluster_id}/overview`.
+
+1. [Create a Log Analytics workspace in Azure Portal](https://learn.microsoft.com/en-us/azure/azure-monitor/logs/quick-create-workspace?tabs=azure-portal).
+
+    1. Navigate to the Log Analytics workspace. Under **Settings**, click **Agents**.
+    1. Click **Log Analytics agent instructions** section header.
+    1. In that section, note the values for **Workspace ID**, **Primary key** (or **Secondary key**)
+
+1. Use one of the following Cloud API commands to enable log export for your CockroachDB {{ site.data.products.dedicated }} cluster. The first presents a basic configuration, where all logs are sent to Azure Monitor using the default settings. The second allows for more detailed customization of the logging configuration, such as the ability to send certain log channels to specific target log tables, or the ability to redact sensitive log entries.
+
+   1. To enable log export for your CockroachDB {{ site.data.products.dedicated }} cluster with default logging configuration, issue the following Cloud API command:
```

Comment:
I would have this as an unordered markdown list `-` and add an anchor link like 

```
- <a id="default-logging-configuration"></a>To enable log export for....
```

Reply:
added Default heading